scripts/restore

```diff
@@ -17,32 +17,13 @@ ynh_script_progression "Restoring the app main directory..."
 
 ynh_restore "$install_dir"
 
-chown -R $app:www-data "$install_dir"
-
 #=================================================
 # RESTORE THE DATA DIRECTORY
 #=================================================
 ynh_script_progression "Restoring the data directory..."
 
 ynh_restore "$data_dir"
 
-#=================================================
-# RESTORE REDIS AND REGENERATE CONFIG
-#=================================================
-
-ynh_script_progression --message="Restore redis and regenerate config..."
-
-# Configure redis
-redis_db=$(ynh_redis_get_free_db)
-admin_email=$(ynh_user_get_info --username=$admin --key=mail)
-email_header=$(get_email_header)
-topest_domain=$(get_topest_domain)
-ynh_app_setting_set --key=redis_db --value="$redis_db"
-
-ynh_config_add --template="default.env" --destination="$install_dir/default.env"
-chmod 400 "$install_dir/default.env"
-chown "$app:$app" "$install_dir/default.env"
-
```

Comment:
These steps are required please, you can't be sure that the redis database number is free when restoring.